ts/components/session/LeftPaneChannelSection.tsx

```diff
@@ -297,11 +295,8 @@ export class LeftPaneChannelSection extends React.Component<Props, State> {
     );
   }
 
-  private handleOnPasteUrl(e: any) {
-    if (e.target.innerHTML) {
-      const cleanText = e.target.innerHTML.replace(/<\/?[^>]+(>|$)/g, '');
```

Comment:
do we have something else where that strip outs tags? or that not a concern?

Reply:
It's a textarea now, so you can type html if you want but it won't be rendered as it, just as plain text.

Reply:
Just wanted to make sure that was on the receiving end as well, so we don't have an XSS here.

Reply:
What we do with this string is test it against this regex:
`const regexURL = /(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?/;
`

and then attempt a connection with that url.